posthog/temporal/data_imports/pipelines/pipeline/utils.py

```diff
@@ -398,6 +400,20 @@ def _update_last_synced_at_sync(schema: ExternalDataSchema, job: ExternalDataJob
     schema.save()
 
 
+# TODO: Implement this, needs to send a transactional email to the members of the team
+def _notify_revenue_analytics_that_sync_has_completed(schema: ExternalDataSchema) -> None:
+    if (
+        schema.source.source_type == ExternalDataSource.Type.STRIPE
+        and schema.source.revenue_analytics_enabled
+        and schema.source.created_at < datetime.now() - datetime.timedelta(minutes=5)
+        and schema.name == STRIPE_DATA_WAREHOUSE_CHARGE_IDENTIFIER
```

Comment:
Just checked out this `STRIPE_DATA_WAREHOUSE_CHARGE_IDENTIFIER` value - it may be better placed somewhere like `posthog/temporal/data_imports/pipelines/stripe/settings.py` - e.g. close to the actual source so that of the schema name does change, it'll be obvious that this would need updating too, but this is very much a nit, so feel free to ignore

Reply:
Ok, great call, will have a PR making that better